jbmc/regression/jbmc/multidimensional-array-char/My.java

```diff
@@ -0,0 +1,17 @@
+public class My {
+  public int check2DCharArray(char[][] board, boolean assertionCheck) {
+    int diff = 0;
+    for (int row = 0; row < 2; row++) {
+      for (int col = 0; col < 2; col++) {
+        if (board[row][col] == 'O') {
+          diff++;
+          assert !assertionCheck;
+        } else if (board[row][col] == 'X') {
+          diff--;
+          assert !assertionCheck;
+        }
+      }
+    }
+    return 0;
```

Comment:
:pick: doesn't really matter, but I would prefer if the function was returning `diff`